src/commands/force/org/list.ts

```diff
@@ -17,7 +19,7 @@ const messages = Messages.loadMessages('@salesforce/plugin-org', 'list');
 
 export class OrgListCommand extends SfdxCommand {
   public static readonly description = messages.getMessage('description');
-
+  public static readonly examples = messages.getMessage('examples').split(EOL);
```

Comment:
Is this for splitting up the [`description` in `messages/open.json` above](https://github.com/salesforcecli/plugin-org/pull/56/files#diff-1b7491620d62423ab4f9c0cd434d2678f216df0622da8eec99883bdeeeda4bb7R2)?

If so this won't work unless it is running on an OS that uses `\n` line endings. It might be better to just explicitly use `.split('\n')` since that's hardcoded in the `.json` file.

Reply:
I noticed that force:org:list --help output didn't include examples, so I just copied what force:org:open does, which IS displaying the examples.  So is force:org:open also wrong?  

Full disclosure: I'm not a developer, just the tech writer, and don't usually touch real code.  But I looked at this and thought maybe I could fix it by simply copying what another command does.  But if the command I copied from is wrong, well, maybe I should just bow out...